mailbox/event/event-cassandra/src/main/java/org/apache/james/mailbox/events/CassandraEventDeadLettersDAO.java

```diff
@@ -111,6 +119,10 @@ private PreparedStatement prepareSelectInsertionIdsWithGroupStatement(Session se
             .map(row -> EventDeadLetters.InsertionId.of(row.getUUID(INSERTION_ID)));
     }
 
+    Mono<Boolean> containEvents() {
+        return executor.executeReturnExists(containEventsStatement.bind().setFetchSize(1));
```

Comment:
why do we set both limit and setFetchSize?

Reply:
To answer you, i had to research something like this:
```
Basically, 'LIMIT' controls the maximum number of results returned by the query, while the 'setFetchSize()' method controls the amount of data transferred at a time.
```
from [cassandra-java-driver](https://docs.datastax.com/en/developer/java-driver/2.1/faq/)

Reply:
Is there any chance, given you asked for at most one element that `setFetchSize`is of any use?

Reply:
Yes, i agreed with you `setFetchSize` is not very much useful here.

Reply:
Agreed, that's pretty much paranoïd ^^